notifications.bs

```diff
@@ -294,7 +294,7 @@ removed from the <a>list of notifications</a>.
 <h3 id="permissions-integration">Permissions integration</h3>
 
 <p>The Notifications API is a <a>powerful feature</a> which is identified by the
-<a for="powerful feature">name</a> "{{PermissionName/notifications}}". [[!Permissions]]
+<a for="powerful feature">name</a> <dfn export permission>"notifications"</dfn>. [[!Permissions]]
```

Comment:
Hmm, should the `<dfn>` include the quotation marks? Also, this removes `<code>`, no?

Reply:
We've used quotations elsewhere (e.g. ["geolocation"](https://www.w3.org/TR/geolocation/#dfn-geolocation))... but it's not mandatory. However, I think permission names are always used as opaque strings. About code, I guess it can just be wrapped? 

So:
```suggestion
<a for="powerful feature">name</a> <code>"<dfn export permission>notifications</dfn>"</code>. [[!Permissions]]
```

Reply:
The style I want to follow is from https://infra.spec.whatwg.org/#strings which has `<code>` inside the quotation marks.

Reply:
Makes sense - I've updated the PR.